docker-initunlocklnd.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+echo "[lnd_unlock] Waiting 2 seconds for lnd..."
+sleep 2;
+
+LND_DIR="/root/.lnd"
```

Comment:
Shouldn't this be using `$LND_DATA` instead of hard-coded value?
Also the other params(LND_WALLET_DIR, etc) should probably get the full path from args from docker-entrypoint.sh?

Reply:
Thanks, switched to using $LND_DATA